lighthouse-core/audits/seo/robots-txt.js

```diff
@@ -182,7 +182,7 @@ class RobotsTxt extends Audit {
     if (!status) {
       return {
         rawValue: false,
-        debugString: 'Lighthouse was unable to download your robots.txt file',
+        explanation: 'Lighthouse was unable to download your robots.txt file',
```

Comment:
is this a failure case on our part or the site's?

Reply:
depends on what you mean by "our part" it could be because the server aborted the request/bad server/etc or it could be because the network failed, either way it seems to belong in explanation instead of errorMessage since there's nothing we would change code-wise, it's expected behavior

Reply:
I guess this is the nuance we should expose so the user has some recourse (either file a bug on lighthouse or know how to start debugging their site). Really a bug on robots-txt gatherer so fine to leave